dotCMS/src/main/java/com/dotcms/cms/login/LoginServiceAPIFactory.java

```diff
@@ -463,6 +471,7 @@ public void doLogout(final HttpServletRequest request, final HttpServletResponse
             LoginServiceAPI.super.doLogout(request, response);
         }
 
+        @CloseDBIfOpened
         @Override
         public boolean passwordMatch(String password, User user) {
             return LoginFactory.passwordMatch(password, user);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "LoginFactory"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)